js/src/lib/interfaces/Types.ts

```diff
@@ -564,6 +564,17 @@ export const PIPELINE_DATA = ensureRecordOfPipelines({
 		modality: "multimodal",
 		color:    "red",
 	},
+	"document-question-answering": {
+		name:     "Document Question Answering",
+		subtasks: [
+			{
+				type: "document-question-answering",
+				name: "Document Question Answering",
+			},
+		],
+		modality: "multimodal",
+		color:    "blue",
```

Comment:
Let's also add this to avoid it showing in the datasets navigation
```suggestion
		color:    "blue",
		hideInDatasets: true,
```